src/lints/function_export_name_changed.ron

```diff
@@ -0,0 +1,55 @@
+SemverQuery(
+    id: "function_export_name_changed",
+    human_readable_name: "function's export name has changed",
+    description: "A function's ABI name with #[no_mangle] or #[export_name = \"name\"] has changed",
+    required_update: Major,  // Not specified in rust semver doc? most similar is for #[repr] changes: https://doc.rust-lang.org/cargo/reference/semver.html#type-layout
+    reference_link: Some("https://doc.rust-lang.org/reference/abi.html#the-no_mangle-attribute"),  // TODO
+    query: r#"
+    {
+        CrateDiff {
+            baseline {
+                item {
+                    ... on Function {
+                        importable_path {
+                            path @output @tag
+                        }
+
+                        export_name @output @tag
+                    }
+                }
+            }
+            
+            current {
+                item {
+                    ... on Function {
+                        name @output
+
+                        export_name @filter(op: "!=", value: ["%export_name"]) @filter(op: "!=", value: ["$null"])
```

Comment:
This is a failure of our docs, so don't worry about it -- there's a built-in filter operator for this:
```suggestion
                        export_name @filter(op: "!=", value: ["%export_name"]) @filter(op: "is_not_null")
```